### DISCUSSION TITLE
Pulp Installation

### DISCUSSION THREAD

**Discussion by goldmanparker** [coin-or/pulp#363] [Q&A]
I have pip installed pulp at the command line, however, whenever I execute my test script:

import pulp as pl
solver_list = pl.listSolvers()

AttributeError: module 'pulp' has no attribute 'listSolvers'

Can someone help point me in the right direction? I have pip uninstalled pulp and reinstalled, but nothing seems to be solving my issue.

**Comment by pchtsp** [comment]
Hello! you're probably using a different python version from the one you just installed. Try doing something like:
```
import pulp as pl
print(pl.__version__)
```
you should get pulp version 2.3

**Reply by dhunton** [reply]
I get:
AttributeError: module 'pulp.pulp' has no attribute '__version__'

**Reply by pchtsp** [reply]
Can you please share the code you're using?

**Reply by dhunton** [reply]
from pulp import *
print(pulp.__version__)

**Reply by pchtsp** [reply]
well... did you do what my answer actually suggested? : ) 
did you get the `pulp.version` from somewhere in the docs? Because you need to do `pulp.__version__` as far as I understand.

**Reply by pchtsp** [reply]
Ok, I actually saw that the formatting in github has edited your code. I recommend using backticks  (\`) before your text to tell github not to format the code.
You can also do three backticks before and after a block of code. Anyways: your problem is you're not importing pulp as I did. Try copying my code and pasting it (the first line is `import pulp as pl`).

**Comment by goldmanparker** [comment]
I was able to get this situated, thank you!
